dune-project

```diff
@@ -0,0 +1,5 @@
+(lang dune 1.0)
+
+(name coq-devel)
+(version 8.9+alpha)
+
```

Comment:
`dune-project` marks the "root" of the project, it will contain maintainer information etc... when opam package generation is done. Indeed I think we need both for now as we have some global targets that have to live in `dune`, but I am not an expert here.